### ISSUE TITLE
Add support for `marker=*`

### ISSUE THREAD

**Issue by arch0345** [openstreetmap/id-tagging-schema#641]
### OSM Tag(s)

marker=*

### How would you like this tag to see supported?

As a Preset

### Label

Post Marker, Aerial Marker, etc.

### Link to OSM Wiki page

https://wiki.openstreetmap.org/wiki/Key:marker

### Status of the Tag

Approved

### Usage of the tag

60,451

### Replaces other Tag?

pipeline=marker

**Comment by tyrasd** [comment]
These should already be supported… see [`marker.json`](https://github.com/openstreetmap/id-tagging-schema/blob/v5.0.1/data/presets/marker.json) and [`marker/*`](https://github.com/openstreetmap/id-tagging-schema/tree/v5.0.1/data/presets/marker). But I think we could add more presets for dedicated types of markers (e.g. pipeline markers, etc.). And corresponding tag upgrade rules are also missing (except for `power=marker`).

**Comment by tiptoptom** [comment]
Would you accept a PR for a deprecation rule changing `pipeline=marker` to `marker=yes` + `subject=pipeline`?

**Comment by tordans** [comment]
> Would you accept a PR for a deprecation rule

@tiptoptom ideally we have some kind of external source that we can point to for those additions. Something like a proposal (accepted or in relevant use and agreement), a (undisputed) wiki (section), a forum discussion with a clear result (voting?)…

**Comment by tiptoptom** [comment]
Yes, I also just found out that `subject=*` is not in the [proposal](https://wiki.openstreetmap.org/wiki/Proposal:Utility_markers_proposal).
But it's on the [Deprecated features Template](https://wiki.openstreetmap.org/wiki/Template:Deprecated_features) because of this [edit](https://wiki.openstreetmap.org/w/index.php?title=Template%3ADeprecated_features&type=revision&diff=2384251&oldid=2360814).

**Comment by tiptoptom** [comment]
For me, `subject=*` makes sense as an interim solution and there are already 14000 of them.
https://taginfo.openstreetmap.org/tags/subject=pipeline

**Comment by tiptoptom** [comment]
Here is a email thread about this:
https://lists.openstreetmap.org/pipermail/tagging/2022-August/065146.html

**Comment by matkoniecz** [comment]
Support for markers is currently implemented.

I am rejecting mass tag migrations using iD tagging schema presets. If such change with no review makes sense then just run mass edit.

For making new dedicated preset for `marker=aerial` - I am not exactly opposed but it would require new icon making sense, added to one of https://github.com/openstreetmap/id-tagging-schema/blob/main/ICONS.md#where-do-the-icons-come-from .

I will close this issue as implemented already.
